sites/goland-guide/src/gatsby-theme-bulmaio/components/navbar/NavbarLink.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
```

Comment:
What's different from the base version of this?

Reply:
I changed how the `className` is computed.
Before: `<i data-testid={`navbarlink-i`} className={`fab fa-lg fa-${icon}`} />`
After: `<i data-testid={`navbarlink-i`} className={`${icon}`} />`

This allows me to use different FA styles as not all the icons use `fab`, see [this change](https://github.com/JetBrains/jetbrains_guide/pull/167/files/3b6dde8e2d42dca7908ad1e5d3ce3a2bbf47fd65#diff-b733ecf9d88dbc768a1ec83fc683851a6530fb24737ddb088f0d948232ee9608R69-R78) as an example.

Reply:
Ok, that's fine...I think we might wind up making your way the everybody-way at some point